framework/src/outputs/ICEUpdater.C

```diff
@@ -27,10 +28,16 @@ InputParameters validParams<ICEUpdater>()
   InputParameters params = validParams<AdvancedOutput<Output> >();
   params += AdvancedOutput<Output>::enableOutputTypes("postprocessor");
 
+  // Get an MooseEnum of the available 'execute_on' options
+  MooseEnum networkingTools("asio=0x01 curl=0x02");
```

Comment:
This comment may be incorrect. Also you probably should just use counting integers here instead of bit field integers here unless you really plan to use bitwise operators.